reqbaz/build.gradle

```diff
@@ -49,6 +49,8 @@ repositories {
 }
 
 dependencies {
+    implementation 'org.jetbrains:annotations:20.1.0'
```

Comment:
TODO: Check whether we can remove this after merging. I don't think we need this dependency